appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/gesture/GestureValidator.kt

```diff
@@ -0,0 +1,13 @@
+package com.bumble.appyx.interactions.core.gesture
+
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Rect
+
+typealias GestureValidator = (Offset, Rect) -> Boolean
```

Comment:
I'd probably prefer an `interface` over a `typealias`, and implementations rather than top level `val`s.